src/components/Filters/__snapshots__/Filters.test.tsx.snap

```diff
@@ -0,0 +1,55 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Filters /> DefaultInteractive story renders snapshot 1`] = `
+<div>
+  <button
+    class="clickable-style clickable-style--lg clickable-style--secondary clickable-style--neutral button filters__button button--secondary"
+    data-bootstrap-override="clickable-style-secondary"
+    type="button"
+  >
+    <svg
+      aria-hidden="true"
+      class="icon"
+      fill="currentColor"
+      height="1.5rem"
+      style="--icon-size: 1.5rem;"
+      width="1.5rem"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <use
+        xlink:href="test-file-stub#filter-list"
+      />
+    </svg>
+    Filters 
+  </button>
+</div>
+`;
+
+exports[`<Filters /> FiltersDrawerComponent story renders snapshot 1`] = `null`;
+
+exports[`<Filters /> FiltersDrawerComponentNoLegend story renders snapshot 1`] = `null`;
```

Comment:
what's up w these?

Reply:
Let me investigate

Reply:
Turns out this is happening due to the usage of the `react-portal` library to append the drawer on the `document`. There is a solution for snaps to happen, which I will add for now, but since there is a native [React Portal](https://reactjs.org/docs/portals.html) I will move the native solution in the upcoming pr

Reply:
Did some pairing with Andrew, cleaned up Drawer component and removed some unnecessary states and snaps work now. Still going to work on removing `react-portal` tho